### PR TITLE
NO-JIRA: restore rosa exceptions

### PR DESCRIPTION
**Pull request by dgoodwin**
Unrevert of #29271 with an exception layered in.

### CODE REVIEW

**Comment by dgoodwin** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-aws-ovn-upgrade

**Comment by dgoodwin** [comment]
/payload-aggregate periodic-ci-openshift-release-master-ci-4.18-upgrade-from-stable-4.17-e2e-aws-ovn-upgrade 10

File: test/extended/operators/resources.go [hunk 1]
```diff
@@ -66,6 +66,8 @@ var _ = g.Describe("[sig-arch] Managed cluster", func() {
 			// Managed service pods that have limits but not requests
 			"apps/v1/Deployment/openshift-monitoring/configure-alertmanager-operator/container/configure-alertmanager-operator/limit[cpu]":    "https://issues.redhat.com/browse/OSD-21708",
 			"apps/v1/Deployment/openshift-monitoring/configure-alertmanager-operator/container/configure-alertmanager-operator/limit[memory]": "https://issues.redhat.com/browse/OSD-21708",
+			"apps/v1/DaemonSet/openshift-multus/cni-sysctl-allowlist-ds/container/kube-multus-additional-cni-plugins/request[cpu]":            "https://issues.redhat.com/browse/TRT-1871",
+			"apps/v1/DaemonSet/openshift-multus/cni-sysctl-allowlist-ds/container/kube-multus-additional-cni-plugins/request[memory]":         "https://issues.redhat.com/browse/TRT-1871",
```
Comment (stbenjam) [inline test/extended/operators/resources.go:70]: Nit, I would separate this from the group of managed services pods with a line break

**Pull request merged**